src/index.html

```diff
@@ -1,12 +1,221 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="header">
+      <a class="header__logo" href="#">
+        <img
+          src="./images/logo.png"
+          alt="logo"
+        >
+      </a>
+      <nav class="header__nav nav">
+        <ul class="nav__list">
+          <li class="nav__item"><a class="nav__link is-active" href="#">Apple</a></li>
+          <li class="nav__item"><a class="nav__link" href="#">Samsung</a></li>
+          <li class="nav__item"><a class="nav__link" href="#">Smartphones</a></li>
+          <li class="nav__item">
+            <a
+              data-qa="nav-hover"
+              class="nav__link"
+              href="#"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item"><a class="nav__link" href="#">Gadgets</a></li>
+          <li class="nav__item"><a class="nav__link" href="#">Tablets</a></li>
+          <li class="nav__item"><a class="nav__link" href="#">Photo</a></li>
+          <li class="nav__item"><a class="nav__link" href="#">Video</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="page__catalog catalog">
+      <div class="card" data-qa="card">
+        <div class="card__content">
+          <div class="card__image"></div>
+          <div class="card__title">APPLE A1419 iMac 27" Retina<br> 5K Monoblock (MNED2UA/A)</div>
+          <div class="card__code">Product code: 195434</div>
+          <div class="card__rate">
+            <div class="stars card__stars">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <div class="card__reviewers">Reviews: 5</div>
+          </div>
+          <div class="card__price"><span class="card__price-text">Price:</span><span class="card__price-number">$2,199</span></div>
```

Comment:
```suggestion
          <div class="card__price">
            <p class="card__price-text">Price:</p>

            <p class="card__price-number">$2,199</p>
          </div>
```
it's easier to read if it's formatted like this :arrow_up: (when you have nested tags)

Also, it's better to use semantic `p` for text (span is not semantic tag)